build.gradle

```diff
@@ -2872,6 +2872,19 @@ project(":controls") {
     addValidateSourceSets(project, sourceSets)
 }
 
+// Add a project declaration for each incubator module here, leaving the
+// incubator placeholder lines as an example.
+// BEGIN: incubator placeholder
```

Comment:
Q: maybe we should mention the JBS every time we say the word "incubator"?

It might be useful to anyone who is looking at the code and has no access to git history (or when git history is obscured by a move).  I mean, JBS is our knowledge base, and it usually helps.

Reply:
I generally prefer not to do this. In this case, the JBS Enhancement isn't what we would want to point to anyway (it would just be a placeholder). I can see some value in pointing to the CSR, so I'll add that here (I don't think there a need to repeat it).

What would really be useful, though, is a pointer to a template with a sample module, along the lines of what I mentioned in #1617, and a pointer to the JEP (which gets back to an earlier discussion of a permanent repository for JEPs). I'll file a follow-up issue to address this.

Reply:
thank you, the CSR is definitely better.

Referring to a ticket has another positive side effect: it's easy to grep.